server/controllers/Core.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var Core = require('../service/CoreService');
+var responseBuilder = require('onf-core-model-ap/applicationPattern/rest/server/ResponseBuilder');
+var responseCodeEnum = require('onf-core-model-ap/applicationPattern/rest/server/ResponseCode');
+var oamLogService = require('onf-core-model-ap/applicationPattern/services/OamLogService');
+
+module.exports.getControlConstruct = async function getControlConstruct(req, res, next) {
```

Comment:
While running the code, I got a 500 response code. I hope you have an issue for this. Please let me if you have one. Thanks. 
Sharing the error for our reference.
<details>
<summary>
Response Error
</summary>

{
  "code": 500,
  "message": ".response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0] should have required property 'operation-server-interface-1-0:operation-server-interface-pac', .response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0]['http-server-interface-1-0:http-server-interface-pac']['http-server-interface-capability']['data-update-period'] should be equal to one of the allowed values: http-server-interface-1-0:DATA_UPDATE_PERIOD_TYPE_REAL_TIME, http-server-interface-1-0:DATA_UPDATE_PERIOD_TYPE_1H_PERIOD, http-server-interface-1-0:DATA_UPDATE_PERIOD_TYPE_24H_PERIOD, http-server-interface-1-0:DATA_UPDATE_PERIOD_TYPE_MANUAL, .response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0] should have required property 'tcp-server-interface-1-0:tcp-server-interface-pac', .response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0] should have required property 'operation-client-interface-1-0:operation-client-interface-pac', .response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0] should have required property 'elasticsearch-client-interface-1-0:elasticsearch-client-interface-pac', .response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0] should have required property 'http-client-interface-1-0:http-client-interface-pac', .response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0] should have required property 'tcp-client-interface-1-0:tcp-client-interface-pac', .response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0] should match exactly one schema in oneOf",
  "errors": [
    {
      "path": ".response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0].operation-server-interface-1-0:operation-server-interface-pac",
      "message": "should have required property 'operation-server-interface-1-0:operation-server-interface-pac'",
      "errorCode": "required.openapi.validation"
    },
    {
      "path": ".response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0]['http-server-interface-1-0:http-server-interface-pac']['http-server-interface-capability']['data-update-period']",
      "message": "should be equal to one of the allowed values: http-server-interface-1-0:DATA_UPDATE_PERIOD_TYPE_REAL_TIME, http-server-interface-1-0:DATA_UPDATE_PERIOD_TYPE_1H_PERIOD, http-server-interface-1-0:DATA_UPDATE_PERIOD_TYPE_24H_PERIOD, http-server-interface-1-0:DATA_UPDATE_PERIOD_TYPE_MANUAL",
      "errorCode": "enum.openapi.validation"
    },
    {
      "path": ".response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0].tcp-server-interface-1-0:tcp-server-interface-pac",
      "message": "should have required property 'tcp-server-interface-1-0:tcp-server-interface-pac'",
      "errorCode": "required.openapi.validation"
    },
    {
      "path": ".response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0].operation-client-interface-1-0:operation-client-interface-pac",
      "message": "should have required property 'operation-client-interface-1-0:operation-client-interface-pac'",
      "errorCode": "required.openapi.validation"
    },
    {
      "path": ".response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0].elasticsearch-client-interface-1-0:elasticsearch-client-interface-pac",
      "message": "should have required property 'elasticsearch-client-interface-1-0:elasticsearch-client-interface-pac'",
      "errorCode": "required.openapi.validation"
    },
    {
      "path": ".response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0].http-client-interface-1-0:http-client-interface-pac",
      "message": "should have required property 'http-client-interface-1-0:http-client-interface-pac'",
      "errorCode": "required.openapi.validation"
    },
    {
      "path": ".response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0].tcp-client-interface-1-0:tcp-client-interface-pac",
      "message": "should have required property 'tcp-client-interface-1-0:tcp-client-interface-pac'",
      "errorCode": "required.openapi.validation"
    },
    {
      "path": ".response['core-model-1-4:control-construct']['logical-termination-point'][19]['layer-protocol'][0]",
      "message": "should match exactly one schema in oneOf",
      "errorCode": "oneOf.openapi.validation"
    }
  ]
}
</details>

Reply:
Yes Prathiba, we have documented few issues in ALAP regarding CC 500 error. Please find the list below:
https://github.com/openBackhaul/AutomatedLinkAcceptanceProxy/issues/107
https://github.com/openBackhaul/AutomatedLinkAcceptanceProxy/issues/106
https://github.com/openBackhaul/AutomatedLinkAcceptanceProxy/issues/108